tests/test_RestartSerialization.cpp

```diff
@@ -112,11 +112,11 @@ TEST_FOR_TYPE(SimulatorReport)
 TEST_FOR_TYPE(SimulatorReportSingle)
 TEST_FOR_TYPE(SimulatorTimer)
 
-namespace Opm { using ATE = AdaptiveTimeSteppingEbos<Properties::TTag::EbosTypeTag>; }
-TEST_FOR_TYPE_NAMED_OBJ(ATE, AdaptiveTimeSteppingEbosHardcoded, serializationTestObjectHardcoded)
-TEST_FOR_TYPE_NAMED_OBJ(ATE, AdaptiveTimeSteppingEbosPID, serializationTestObjectPID)
-TEST_FOR_TYPE_NAMED_OBJ(ATE, AdaptiveTimeSteppingEbosPIDIt, serializationTestObjectPIDIt)
-TEST_FOR_TYPE_NAMED_OBJ(ATE, AdaptiveTimeSteppingEbosSimple, serializationTestObjectSimple)
+namespace Opm { using ATE = AdaptiveTimeStepping<Properties::TTag::EbosTypeTag>; }
+TEST_FOR_TYPE_NAMED_OBJ(ATE, AdaptiveTimeSteppingHardcoded, serializationTestObjectHardcoded)
+TEST_FOR_TYPE_NAMED_OBJ(ATE, AdaptiveTimeSteppingPID, serializationTestObjectPID)
+TEST_FOR_TYPE_NAMED_OBJ(ATE, AdaptiveTimeSteppingPIDIt, serializationTestObjectPIDIt)
+TEST_FOR_TYPE_NAMED_OBJ(ATE, AdaptiveTimeSteppingSimple, serializationTestObjectSimple)
```

Comment:
Tiny nit, but `ATE` probably isn't a good name anymore once we remove the `Ebos` part of the class name.  How about `ATS` instead?

Reply:
okay